awx/ui_next/src/screens/Setting/Settings.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Link, Route, Switch } from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { PageSection, Card } from '@patternfly/react-core';
+import ContentError from '../../components/ContentError';
+import Breadcrumbs from '../../components/Breadcrumbs';
+import ActivityStream from './ActivityStream';
+import AzureAD from './AzureAD';
+import GitHub from './GitHub';
+import GoogleOAuth2 from './GoogleOAuth2';
+import Jobs from './Jobs';
+import LDAP from './LDAP';
+import License from './License';
+import Logging from './Logging';
+import MiscSystem from './MiscSystem';
+import Radius from './Radius';
+import SAML from './SAML';
+import SettingList from './SettingList';
+import TACACS from './TACACS';
+import UI from './UI';
+
+function Settings({ i18n }) {
+  const breadcrumbConfig = {
+    '/settings': i18n._(t`Settings`),
+    '/settings/activity_stream': i18n._(t`Activity stream`),
+    '/settings/azure': i18n._(t`Azure AD`),
+    '/settings/github': i18n._(t`GitHub`),
+    '/settings/google_oauth2': i18n._(t`Google OAuth2`),
+    '/settings/jobs': i18n._(t`Jobs`),
+    '/settings/ldap': i18n._(t`LDAP`),
+    '/settings/license': i18n._(t`License`),
+    '/settings/logging': i18n._(t`Logging`),
+    '/settings/miscellaneous_system': i18n._(t`Miscellaneous system`),
+    '/settings/radius': i18n._(t`Radius`),
+    '/settings/saml': i18n._(t`SAML`),
+    '/settings/tacacs': i18n._(t`TACACS+`),
+    '/settings/user_interface': i18n._(t`User interface`),
+  };
+
+  return (
+    <>
+      <Breadcrumbs breadcrumbConfig={breadcrumbConfig} />
+      <Switch>
+        <Route path="/settings/activity_stream">
+          <ActivityStream />
+        </Route>
+        <Route path="/settings/azure">
+          <AzureAD />
+        </Route>
+        <Route path="/settings/github">
+          <GitHub />
+        </Route>
+        <Route path="/settings/google_oauth2">
+          <GoogleOAuth2 />
+        </Route>
+        <Route path="/settings/jobs">
+          <Jobs />
+        </Route>
+        <Route path="/settings/ldap">
+          <LDAP />
+        </Route>
+        <Route path="/settings/license">
```

Comment:
You may have already discussed this w/ folks but we probably need to hide the license routes, list items, pages, etc. for non-product installs.

I don't know off the top of my head how the ui-classic does it, but one way to check is by making a request to `/api/v2/config/` and seeing if the license type is "OPEN":

![Screenshot from 2020-07-28 19-24-09](https://user-images.githubusercontent.com/9753817/88739619-f1821880-d108-11ea-9baa-0a99f6816600.png)